### PR TITLE
Release 7.101.0.1

### PR DESCRIPTION
**Pull request by samsymons**
Please make sure all GH checks passed before merging. It can take around 20 minutes.
Briefly review this PR to see if there are no issues or red flags and then merge it.

### CODE REVIEW

**Comment by samsymons** [comment]
This change fixes IPv6 connectivity and improves pixels ahead of the NetP waitlist test.

File: DuckDuckGo/SettingsViewController.swift [hunk 1]
```diff
@@ -127,7 +127,8 @@ class SettingsViewController: UITableViewController {
     private lazy var shouldShowNetPCell: Bool = {
 #if NETWORK_PROTECTION
         if #available(iOS 15, *) {
-            return featureFlagger.isFeatureOn(.networkProtection)
+            let accessController = NetworkProtectionAccessController()
```
Comment (graeme) [inline DuckDuckGo/SettingsViewController.swift:130]: Heads up @afterxleep

**Comment by graeme** [comment]
@samsymons Tried to build an AdHoc build but ended up with an invalid bundle. Not sure if that’s something worth looking into...

**Review by graeme — APPROVED** [review]

File: DuckDuckGo/SettingsViewController.swift (hunk 1 above)
Comment (afterxleep) [reply DuckDuckGo/SettingsViewController.swift:130]: oh!, no more feature flagging @graeme?  Here's where this logic has moved:

https://github.com/duckduckgo/iOS/pull/2255/files#diff-ad06089c79361f33a5f83335daca81941fcb6ec2fda955c52a8783b1a551ba61R49

Should we kill the feature flag check from here?

https://github.com/duckduckgo/iOS/pull/2255/files#diff-2e2f08fd3299395ca1e3d3fdf69f575a8bbc576bd3e5a46374975f85ebe410bdR84

**Pull request merged**